### PR TITLE
Reviewer MIRW: Don't crash

### PR DESCRIPTION
**Pull request by graemerobertson**

### CODE REVIEW

File: src/snmp_infinite_timer_count_table.cpp [hunk 1]
```diff
@@ -206,6 +197,16 @@ namespace SNMP
...
         }
 
+        new_oid_len += 2;
+        new_oid[ROOT_OID_LEN + 1 + tag.length()] = identifier.at(0);
+        new_oid[ROOT_OID_LEN + 2 + tag.length()] = identifier.at(1);
+
+        char buf1[64];
+        snprint_objid(buf1, sizeof(buf1),
+                      new_oid, new_oid_len);
+        TRC_DEBUG("Parsed SNMP request to OID %s with tag %s and identifier <%d, %d>",
```
Comment (mirw) [inline src/snmp_infinite_timer_count_table.cpp]: Should we still trace something earlier?  Maybe there's not much we can trace?

**Comment by mirw** [comment]
Looks good.  One comment inline.  Presumably you've tested both `snmpget` and `snmpgetnext` live?

**Comment by graemerobertson** [comment]
@rkd-msw what do you think we should do with this? I'd like to merge it

**Comment by rkd-msw** [comment]
Agreed.

**Pull request merged**